### ISSUE TITLE
Borderlands doesn't open

### ISSUE THREAD

**Issue by ArmandoAmmar** [c0dycode/BL-AutoexecDLL#12]
When I installed this, borderland stoped to open if you need more informations, please tell me what you need and I'll try to provide.

**Comment by c0dycode** [comment]
This is most likely due to the most recent updates.
Currently I have no ETA on when I'll be able to look into it though.

**Comment by daveamol** [comment]
Yes it looks like the initial launcher shows up but after that the game just doesn't open or crashes. No BL2 splash screen or anything. Please tell me if I can share any logs or test out something

**Comment by daveamol** [comment]
Attaching the .dmp file I got after the game fails to start
https://filebin.net/cmqdq5e0wtjdc2dm
